### ISSUE TITLE
Feature Request: Include Thera connections for calculating the shortest route!

### ISSUE THREAD

**Issue by seenkley** [Slazanger/SMT#68]
Hey!
First off, this tool is a blast - I used it on a daily basis since I discovered it 💪 

I miss one feature though, or at least I did not find it yet. Is it possible to include Thera connections when calculating routes ?

**Comment by Boot103** [comment]
Yes please this would be awesome. Maybe in the thera page have closest connection to you. This would be so useful.

**Comment by tehraven** [comment]
Could be just a quick check to calculate `((distance between start and thera) + (distance between end and thera))` and if that's shorter than the normal autopilot route, just set the thera systems as waypoints. 

Might confuse people if they just absent-mindedly carryon from there and don't go straight to the thera hole. AFAIK there's no way to check if the person has the EVE Scout folder activated, nor an ESI means to disable autopilot at the first waypoint system.
